src/main/kotlin/org/jitsi/jibri/service/impl/FileRecordingJibriService.kt

```diff
@@ -142,16 +142,27 @@ class FileRecordingJibriService(
             logger.info("Selenium joined the call, starting the capturer")
             capturer.start(sink)
         }
-        jibriSelenium.addToPresence("session_id", fileRecordingParams.sessionId)
-        jibriSelenium.addToPresence("mode", JibriIq.RecordingMode.FILE.toString())
-        jibriSelenium.sendPresence()
+        try {
+            jibriSelenium.addToPresence("session_id", fileRecordingParams.sessionId)
+            jibriSelenium.addToPresence("mode", JibriIq.RecordingMode.FILE.toString())
+            jibriSelenium.sendPresence()
+        } catch (t: Throwable) {
+            logger.error("Error while setting fields in presence", t)
+            publishStatus(ComponentState.Error(ErrorScope.SESSION, "Unable to set presence values"))
+        }
     }
 
     override fun stop() {
         logger.info("Stopping capturer")
         capturer.stop()
         logger.info("Quitting selenium")
-        val participants = jibriSelenium.getParticipants()
+        val participants = try {
+            jibriSelenium.getParticipants()
+        } catch (t: Throwable) {
+            logger.error("An error occurred while trying to get the participants list, proceeding with " +
+                    "and empty participants list", t)
```

Comment:
*an empty. Is it okay to continue with an empty list? Does that trigger the logic for auto-exiting?